tests/elis_api_client/test_documents.py

```diff
@@ -11,6 +14,26 @@ def file_data() -> bytes:
         return f.read()
 
 
+@pytest.fixture
+def dummy_document() -> dict:
```

Comment:
I'd use the `Document` model here and return a `Document` object instead of dict.

Reply:
I was thinking about it too but the rest of the tests is using this approach so at the end I stick with consistency. :thinking: